packages/OcLaunch/OcLaunch.0.1.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+name: "OcLaunch"
+version: "0.1.3"
+maintainer: "Joly Clément <leowzukw@vmail.me>"
+authors: [ "Joly Clément <leowzukw@vmail.me>" ]
+license: "CeCILL"
+homepage: "http://www.oclaunch.tuxfamily.org"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+depends: [
+  "ocamlbuild"
+  "oasis"
```

Comment:
It is not. [oasis2opam](https://github.com/ocaml/oasis2opam) only generates the dependencies below.